Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml

```diff
@@ -261,3 +261,40 @@
     steps: 5
     zeroVisible: true
   - type: Appearance
+
+- type: entity
+  parent: BaseWeaponBattery
+  id: WeaponHealGun
+  name: Medi-Cannon
```

Comment:
should call it medibeam and if not medi-cannon for consistence with everything else